### ISSUE TITLE
Update "Keyboard focus has a low contrast - is this a WCAG failure?"

### ISSUE THREAD

**Issue by Jefferydo** [cenw-wscoe/WCAG-in-Plain-Language#62]
The answer is "depends":
"If the keyboard focus indicator was altered using CSS proprety outline-color on the interactive element, yes this is a failure. However if the keyboard focus is the browser's default outline, this is not a failure."

It doesn't matter how the keyboard focus indicator contrast is set, via color, outline-color, border-color, background-color or any browser default. If it fails a contrast check, it's a WCAG failure.

Course page on [Contrast](https://bati-itao.github.io/learning/esdc-self-paced-web-accessibility-course/module7/contrast.html).
